### PR TITLE
Added network interface counters

### PR DESCRIPTION
**Pull request by devlife**
Added:

- `\Network Interface(*)\Bytes Received/sec`
- `\Network Interface(*)\Bytes Sent/sec`
- `\Network Interface(*)\Current Bandwidth`

Additionally there were special characters in this file that git didn't like so I replaced those as well.

### CODE REVIEW

File: PerfCounterReporter/CounterDefinitions/system.counters [hunk 1]
```diff
@@ -81,14 +81,30 @@
 \PhysicalDisk(*)\Avg. Disk sec/Read
 
 #The number of bytes sent and received over a specific network adapter, including
-# framing characters. Be sure to record the throughput of your SQL Server�s NIC 
-#card(s).  Watch for this value possibly exceeding the NIC�s specifications, 
+# framing characters. Be sure to record the throughput of your SQL Serverss NIC
```
Comment (tedoc2000) [inline PerfCounterReporter/CounterDefinitions/system.counters]: Lost a ` on Servers'

File: PerfCounterReporter/CounterDefinitions/system.counters [hunk 2]
```diff
@@ -81,14 +81,30 @@
 \PhysicalDisk(*)\Avg. Disk sec/Read
 
 #The number of bytes sent and received over a specific network adapter, including
-# framing characters. Be sure to record the throughput of your SQL Server�s NIC 
-#card(s).  Watch for this value possibly exceeding the NIC�s specifications, 
+# framing characters. Be sure to record the throughput of your SQL Serverss NIC 
+#card(s).  Watch for this value possibly exceeding the NICss specifications,
```
Comment (tedoc2000) [inline PerfCounterReporter/CounterDefinitions/system.counters]: Same here with NICs

File: PerfCounterReporter/CounterDefinitions/system.counters [hunk 3]
```diff
@@ -81,14 +81,30 @@
...
 #The number of bytes sent and received over a specific network adapter, including
-# framing characters. Be sure to record the throughput of your SQL Server�s NIC 
-#card(s).  Watch for this value possibly exceeding the NIC�s specifications, 
+# framing characters. Be sure to record the throughput of your SQL Serverss NIC 
+#card(s).  Watch for this value possibly exceeding the NICss specifications, 
 #especially when conducting large and/or multiple backups or copies to network drives.  
 #A high-speed network and/or a NIC dedicated to admin processes often alleviates this 
-#bottleneck. This counter is a sum of �Network Interface\\Bytes Received/sec� and 
-#�Network Interface\\Bytes Sent/sec�.  In some situations, you may wish to determine 
+#bottleneck. This counter is a sum of `Network Interface\\Bytes Received/sec` and
```
Comment (tedoc2000) [inline PerfCounterReporter/CounterDefinitions/system.counters]: Just use regular double quotes

File: PerfCounterReporter/CounterDefinitions/system.counters [hunk 4]
```diff
@@ -81,14 +81,30 @@
...
-# framing characters. Be sure to record the throughput of your SQL Server�s NIC 
-#card(s).  Watch for this value possibly exceeding the NIC�s specifications, 
+# framing characters. Be sure to record the throughput of your SQL Serverss NIC 
+#card(s).  Watch for this value possibly exceeding the NICss specifications, 
 #especially when conducting large and/or multiple backups or copies to network drives.  
 #A high-speed network and/or a NIC dedicated to admin processes often alleviates this 
-#bottleneck. This counter is a sum of �Network Interface\\Bytes Received/sec� and 
-#�Network Interface\\Bytes Sent/sec�.  In some situations, you may wish to determine 
+#bottleneck. This counter is a sum of `Network Interface\\Bytes Received/sec` and 
+#`Network Interface\\Bytes Sent/sec`.  In some situations, you may wish to determine
```
Comment (tedoc2000) [inline PerfCounterReporter/CounterDefinitions/system.counters]: Same here

File: PerfCounterReporter/CounterDefinitions/system.counters (hunk 1 above)
Comment (devlife) [inline PerfCounterReporter/CounterDefinitions/system.counters]: That's weird. I added them again.

**Comment by tedoc2000** [comment]
Thanks looks great

**Comment by devlife** [comment]
Awesome thanks!

**Pull request merged**